tests/common/gllayerrenderer.cpp

```diff
@@ -66,11 +67,27 @@ bool GLLayerRenderer::Init(uint32_t width, uint32_t height, uint32_t format,
   glBindFramebuffer(GL_FRAMEBUFFER, gl_framebuffer_);
   glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0,
                             GL_RENDERBUFFER, gl_renderbuffer_);
+
   if (glGetError() != GL_NO_ERROR) {
     printf("failed to create GL framebuffer\n");
     return false;
   }
+#else
+  glGenTextures(1, &gl_renderbuffer_);
```

Comment:
yes, i tried to use texture buffer  for texture, but the layer shakes always, so i keep the debug code here and, actually, the code isn't used, i will remove the debug code here. thanks